src/content/peerconnection/bandwidth/js/main.js

```diff
@@ -185,6 +195,26 @@ bandwidthSelector.onchange = function() {
   bandwidthSelector.disabled = true;
   var bandwidth = bandwidthSelector.options[bandwidthSelector.selectedIndex]
       .value;
+
+  if (adapter.browserDetails.browser === 'chrome' &&
```

Comment:
should probably add a comment here that explains what this is doing and why its chrome-only